presto-orc/src/main/java/com/facebook/presto/orc/metadata/CompressionKind.java

```diff
@@ -15,5 +15,24 @@
 
 public enum CompressionKind
 {
-    NONE, ZLIB, SNAPPY, LZ4, ZSTD
+    NONE(0),
+    ZLIB(9),
+    SNAPPY(17),
+    LZ4(14),
+    ZSTD(21);
+
+    /**
+     * Minimum size of the input data that would benefit from compression.
+     */
+    private final int minCompressibleSize;
```

Comment:
I had to add it to CompressionKind instead of Compressor because we are using airlift compressor interfaces.